### PR TITLE
Update container base to Ubuntu 20.04, fix lint, update go to v19

### PR DESCRIPTION
**Pull request by redhug1**
### What

As per title + fix audit.

### How to review

Check container image is available.

### Who can review

Anyone

### CODE REVIEW

File: handlers/handlers.go [hunk 1]
```diff
@@ -63,15 +63,15 @@ func isProtectedCookie(stringToFind string) bool {
 func removeNonProtectedCookies(w http.ResponseWriter, req *http.Request) {
 	for _, cookie := range req.Cookies() {
 		if !isProtectedCookie(cookie.Name) {
-			cookie := &http.Cookie{
+			cookie2 := &http.Cookie{
```
Comment (mr-nick17) [inline handlers/handlers.go:66]: I assume you're renaming this as the linter has complained that variable name is the same as the value? However `cookie2` is not a great time; I suggest either `setCookie` or pass through a slice 
```go
			http.SetCookie(w, &http.Cookie{
				Name:     cookie.Name,
				Value:    "",
				Path:     "/",
				Expires:  time.Unix(0, 0),
				MaxAge:   0,
				HttpOnly: false,
			})
```

File: handlers/handlers_test.go [hunk 2]
```diff
@@ -270,7 +271,7 @@ func initialiseMockCookiesPageModel(cfg *config.Config, policy cookies.Policy, i
 	page.SiteDomain = cfg.SiteDomain
 	page.PatternLibraryAssetsPath = cfg.PatternLibraryAssetsPath
 	page.PreferencesUpdated = hasSetPreference
-	page.FeatureFlags.SixteensVersion = "67f6982"
+	page.FeatureFlags.SixteensVersion = mapper.SixteensVersionStr
```
Comment (mr-nick17) [inline handlers/handlers_test.go:274]: No value in setting this to a constant from a different package. I would argue that func `initialiseMockCookiesPageModel` is a mapper test and not needed in `handlers`

File: config/config.go [hunk 3]
```diff
@@ -22,17 +22,17 @@ var cfg *Config
 
 // Get returns the default config with any modifications through environment variables
 func Get() (*Config, error) {
-	cfg, err := get()
+	config, err := get()
```
[the remark this answers is not in the document]
Comment (redhug1) [reply config/config.go:25]: The linter only complains about it here, feel free to pull my feature branch, undo this change and run 'make lint' and you will see what the problem is.
The change is needed for the linter to pass, and thus for CI to play ball.

File: handlers/handlers.go (hunk 1 above)
Comment (redhug1) [reply handlers/handlers.go:66]: Thanks, i'll go with 'setCookie'.

File: handlers/handlers_test.go (hunk 2 above)
Comment (redhug1) [reply handlers/handlers_test.go:274]: My change undone.

**Review by mr-nick17 — APPROVED** [review]

**Pull request merged**